tests/vite-app/vite.config.mjs

```diff
@@ -42,7 +43,13 @@ export default defineConfig({
     },
   },
   build: {
+    outDir: resolve(process.cwd(), "dist"),
     rollupOptions: {
+      plugins: [
+        rollupResolve({
+          extensions: [".gjs", ".js", ".hbs", ".ts", ".gts"],
+        }),
+      ],
```

Comment:
I'm pretty sure that we can't add this plugin and expect things to work 🤔 even if we needed to we should only add this in a new PR. Can you remove it from this one?

Reply:
the build fails without this

Reply:
or it did, its working now without it...